packages/facility-server/app/routes/apiv1/patient/patientProgramRegistration.js

```diff
@@ -271,6 +280,34 @@ patientProgramRegistration.post(
   }),
 );
 
+patientProgramRegistration.put(
+  '/:patientId/programRegistration/:programRegistryId/condition/:conditionId',
```

Comment:
I wasn't sure if this endpoint should be something like `programRegistrationCondition/:conditionId' since it doesn't really need to know about the programRegistryId instead but I went for the longer url for consistency